webhooks-extension/src/components/WebhookDisplayTable/WebhookDisplayTable.js

```diff
@@ -191,28 +342,36 @@ export class WebhookDisplayTable extends Component {
               )}
               />
           </div>
+
+          <div className="modal">
+            <Modal open={this.state.showDeleteDialog}
+              className=".delete-modal"
+              modalLabel="Are you sure?"
+              modalHeading="Really delete webhook(s)?"
+              primaryButtonText="Delete webhook"
```

Comment:
I would just go with Delete and Cancel as the labels on the buttons

Reply:
Done, and modified tests accordingly